analysers/analyser_merge_bicycle_parking_FR_paris.py

```diff
@@ -48,8 +48,10 @@ def __init__(self, config, logger = None):
         self.init(
             u"https://opendata.paris.fr/explore/dataset/stationnement-voie-publique-emplacements/information/",
             u"Stationnement sur voie publique - emplacements vélos",
-            CSV(Source(attribution = u"Ville de Paris", millesime = "02/2018",
-                    fileUrl = u"https://opendata.paris.fr/explore/dataset/stationnement-voie-publique-emplacements/download/?format=csv&timezone=Europe/Berlin&use_labels_for_header=true"),
+            CSV(SourceOpenDataSoft(attribution = u"Ville de Paris",
```

Comment:
When we edit code, u-string can be removed in favour of string.

Reply:
That's what I did to my edited strings. I did the least modification possible and followed [the contribution guide](https://github.com/osm-fr/osmose-backend/blob/master/CONTRIBUTING.md#coding-style).

But I'll clean a bit more around my edits :+1:

Reply:
I've done most u-string deletion in a separate commit.